### DISCUSSION TITLE
Command and Option

### DISCUSSION THREAD

**Discussion by BraxtonMDevelops** [d12frosted/homebrew-emacs-plus#480] [Q&A] [answered]
Hi there! It seems command and option are swapped by default, while this isn't an issue in and of itself, I was wondering if a) I can revert this change, and b) why it seemingly isn't mentioned in the readme. Thanks so much!

EDIT: I realized expected behavior here is unclear. I expected command to act as `M-` and Option does so currently.

**Comment by d12frosted** [comment, marked as answer]
Hey @BraxtonMDevelops,

I think your epxectation is reverted 😸 Option is meta key by convention. And not only in Emacs. Luckily, this behaviour is controlled by `mac-option-modifier` and `mac-command-modifier` variables. I also like to use command as meta key in Emacs, so I have the following lines in my `init.el`. 

```
(setq mac-option-modifier nil
        mac-command-modifier 'meta)
```

And why it's not mentioned... Well, this project is just a homebrew formula that installs GNU Emacs. Explaining how to configure  Emacs is not the goal of this project. And also, Emacs has lots of built-in documentation you can use 😸 

Emacs+ documentation covers only configurations of stuff that comes from custom patches (like `no-title-bar` or system appearance change).

Hope that explains :)

**Reply by BraxtonMDevelops** [reply]
Ah. I'm a very newbie MacOS user. Was unaware that option would be "More correct". I have honestly no idea what Command maps to generally. I just find the placement of it to be better for hitting in the `M-` key sequences. Thanks for your code snippet! as well as the insight!

Thanks for mentioning this.

**Reply by d12frosted** [reply]
You are welcome! The same, I also find the placement of command key better for meta key, especially since many of my key bindings are based on it. In addition, I map Caps Lock to CTRL, so I have no struggles with Emacs key bindings 😄

**Reply by BraxtonMDevelops** [reply]
my struggles are alleviated by the fact that I primarily use evil based configs. no remapping caps lock.
